gatsby-node.js

```diff
@@ -219,6 +223,31 @@ class Plugin {
 			typeName: this.options?.type?.name || 'DirectusData',
 			fieldName: this.options?.type?.field || 'directus',
 			headers: this.headers.bind(this),
+			fetch: async (uri, options) => {
+				function request() {
+					return nodeFetch(uri, options);
+				}
+
+				let count = 0;
+				let response = null;
+				let error = null;
+
+				while (response === null && count++ < this.retries) {
+					try {
+						response = await request();
+					} catch (err) {
+						err = error;
+					}
+
+					if (count > 0) {
+						await sleep(randInt(1000, 8000));
```

Comment:
Should this be random, or should each attempt be slightly longer than the previous one? (Exponential backoff)